packages/bits/src/lib/select-v2/base-select-v2.ts

```diff
@@ -433,4 +444,30 @@ export abstract class BaseSelectV2 implements AfterViewInit, AfterContentInit, C
             resizeObserver.unobserve(this.elRef.nativeElement);
         });
     }
+
+    /**
+     * This helps to dynamically set minHeight for overlay to avoid issues with double
+     * scroll. Overlay minHeight should be bigger than cdkVirtualScroll container.
+     */
+    private adjustDropdownOnVScrollResize(): void {
+        if (!this.cdkVirtualScroll) {
+            return;
+        }
+
+        const element = this.cdkVirtualScroll.elementRef.nativeElement;
+        const height = parseInt(element.style.height, 10);
+        const minHeight = Number.isNaN(height) ? 0 : height + 10;
```

Comment:
Can we use a named constant for the 10 to make it more self-documenting? Maybe `vScrollHeightBuffer`? This constant can also be used at line 465.